1/ToDoApp/src/components/ToDoItem/ToDoItem.spec.js

```diff
@@ -0,0 +1,46 @@
+import { mount } from '@vue/test-utils'
+import ToDoItem from './ToDoItem.vue'
+
+describe('ToDoItem', () => {
+  describe('given a todo', () => {
+    const wrapper = mount(ToDoItem, {
+      propsData: {
+        todo:{
+          text: 'test-todo'
+        }
+      }
+    })
+    it('renders todo.text', () => {
+      expect(wrapper.vm.todo.text).toBe('test-todo');
+    })
+    it('does not show input field', () => {
+      expect(wrapper.vm.editModeOn).toBe(false);
+    })
+    describe('click on Edit Button', () => {
+      const editButton=wrapper.find('#editButton');
```

Comment:
I would not do that outside of a `beforeEach`. You will create non-atomic tests because the `editButton` might be different in some test cases and then execution order matters, which is not what you want.